server/v2/cometbft/mempool/mempool.go

```diff
@@ -0,0 +1,41 @@
+package mempool
+
+import (
+	"context"
+	"errors"
+
+	"cosmossdk.io/core/transaction"
+)
+
+// Mempool defines the required methods of an application's mempool.
+type Mempool[T transaction.Tx] interface {
+	// Insert attempts to insert a Tx into the app-side mempool returning
+	// an error upon failure.
+	Insert(context.Context, T) error
+
+	// Select returns an Iterator over the app-side mempool. If txs are specified,
+	// then they shall be incorporated into the Iterator. The Iterator must be
+	// closed by the caller.
```

Comment:
What happened to me comment here? 😆

Reply:
I assumed this was used to include more txs in the mempool, but I checked the current implementations and we are not using it at all 🤨 So I guess we'd like to remove it?